string file_name
---
string output
